apps/condo/domains/organization/schema/Organization.js

```diff
@@ -78,6 +80,26 @@ const Organization = new GQLListSchema('Organization', {
             type: Json,
             isRequired: false,
         },
+        phone: {
+            schemaDoc: 'Normalized organization phone in E.164 format without spaces',
```

Comment:
Should we move it to constant? It'll be easier to modify phone validations if needed...